stdpopsim/homo_sapiens.py

```diff
@@ -169,7 +169,38 @@ def chromosome_factory(name, genetic_map=None, length_multiplier=1):
 default_generation_time = 25
 
 
-class GutenkunstThreePopOutOfAfrica(models.Model):
+class HomoSapiensModel(models.Model):
+    """
+    TODO: documentation
+    """
+    def __init__(self):
+        super().__init__()
+        self.generation_time = default_generation_time
+        self.default_population_size = 10000
+
+
+class GenericConstantSize(HomoSapiensModel, generics.GenericMixin):
+    """
```

Comment:
No docstring - should be inherited from the Mixin (I'm not sure whether it needs to come first or last for this to happen), and so the online docs and CLI will get the standard documentation.

Reply:
By "it" I mean the order of the parent classes. We want ot inherit the docstring from the Mixin.